### ISSUE TITLE
Cannot launch the console version of the bundle App on OSX

### ISSUE THREAD

**Issue by remyroy** [CleverRaven/Cataclysm-DDA#16302]
When trying to launch `Cataclysm.app/Contents/MacOS/Cataclysm.sh`, the following error is displayed:

```
51:51: syntax error: Expected string but found end of script. (-2741)
```

Pinging @patternoia

**Comment by patternoia** [comment]
@remyroy try changing "Terminal" to "Terminal.app" in this script. 
You probably had other application with the name "Terminal" which does not understand the "do script" command.

If it will work for you, I'll submit a pull request with this small fix.

**Comment by remyroy** [comment]
I just tested this issue again and it seems to work even without changing "Terminal" to "Terminal.app" in the script. It seems like the problem is only happening when the bundle is in a dmg mounted disk and I try to run it from there.

Do you think there could be a link between this issue and the the fact that the dmg mounted disk is read-only?

**Comment by patternoia** [comment]
Might be. I am getting the same error when running it from dmg disk.
Anyway invoking "Terminal.app" from apple script is more correct comparing to "Terminal"
I will file a pull request for this.

**Comment by mugling** [comment]
Fixed by #16309

**Comment by Leland** [comment]
This works on my end! 

Closing due to age and silence on issue. If this is still an active issue, feel free to let me know or open up a new one.
